src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/reader/ReaderMetricsProcessor.java

```diff
@@ -201,8 +219,43 @@ public void trimOldSnapshots() throws Exception {
     trimMap(shardRqMetricsMap, RQ_SNAPSHOTS);
     trimMap(httpRqMetricsMap, HTTP_RQ_SNAPSHOTS);
     trimMap(masterEventMetricsMap, MASTER_EVENT_SNAPSHOTS);
-    trimDatabases(
-        metricsDBMap, MAX_DATABASES, PluginSettings.instance().shouldCleanupMetricsDBFiles());
+
```

Comment:
The better way might be to move lines 223 to 258 in the trimDatabase method. what do you think ?

Reply:
We should keep trimDatabases for now. It's been pretty well tested and it is public. The codebase itself can use some cleanup, so this can be taken care of with that. However, we should retain the code for now in case we want to make use of it down the road.

See https://github.com/opendistro-for-elasticsearch/performance-analyzer-rca/pull/315#discussion_r475793690

Reply:
If we are removing the usage in the code, we should remove the method. Git takes care of preserving history. So, if it is required in future, we can resurrect it.

Reply:
Sure, will remove it.